### ISSUE TITLE
configuration on tsplib data

### ISSUE THREAD

**Issue by jiajinghu19** [yining043/NeuOpt#1]
Hi, I am testing the pretrained TSP100 model on tsplib data. Using the default configuration, on a280 data, it gave a best value of 23508, which has gap of 800% compared to the optimal value. May I ask if there's a different set of configuration I should be using or did I do anything wrong? Thanks!

CUDA_VISIBLE_DEVICES=0,1,2,3 python run.py --eval_only --no_saving --no_tb --init_val_met random  --k 4 --problem tsp --val_m 1 --stall 10 --T_max 1000 --load_path pre-trained/tsp100.pt

**Comment by yining043** [comment]
Many thanks for your interest in our work!

The zero-shot generalization of neural solvers to larger sizes and different node distributions is still a research focus and an important future work especially for L2S solvers. You may directly use the pre-trained TSP model to test larger TSPLIB instances. But for better performance, we may consider the following practices:

1. Use the pre-trained model on size 200 for the A280 instance
2. The x-y coordinates of the instance should be normalized to the range [0,1] (as perused in training) before being fed into the solver
3. Use "--init_val_met greedy" as the initial solution
4. Consider larger 'T_max' and/or larger 'val_m'
5. Consider a re-start strategy that re-initializes the solution as the best-so-far solution if the solver fails to find any better solution for a consecutive number of steps (e.g., 10), similar to that used in Wu et. al.

**Comment by jiajinghu19** [comment]
Thanks for the quick response! I got good results after normalizing coordinates to [0,1] range. We can close this issue

**Comment by yining043** [comment]
Welcome!
